### ISSUE TITLE
*BUG* Custom Breakpoints Active Breaks Responsive Settings on Page Reload

### ISSUE THREAD

**Issue by Northwest-Studio** [elementor/elementor#16940]
Hi,

I have recreated this issue on a number of sites. Issue resolves when custom breakpoints are turned off.

Upon first page load all my styling is fine, column widths etc. Upon refresh the styling breaks and inherits the desktop styles.

Issue reverts once CSS is regenerated, but once again breaks upon page refresh. 

Please see screen recording of bug upon page refresh, reseting page on CSS regeneration, breaking on refresh once again, and then fixing upon deactivating custom breakpoints: https://www.loom.com/share/1325f41293794501a40946a923fd532d

I have the columns set to be 100%W on mobile. Everything looks fine in the editor.

Could this please be investigated? I have had this happen on multiple sites now and have had to manually overwrite with custom CSS to fix on my live sites.

Thanks

## Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

## Isolating the problem

- [ X] This bug happens with only Elementor plugin active (and Elementor Pro).
- [ X] This bug happens with a default WordPress theme active.
- [X ] I can reproduce this bug consistently using the steps above.

## Environment

<details>
<summary>System Info</summary>
```
					
== Server Environment ==
	Operating System: Linux
	Software: Apache/2.4.25 (Debian)
	MySQL version: mariadb.org binary distribution v5.6.49-89.0
	PHP Version: 7.4.25
	PHP Max Input Vars: 10000
	PHP Max Post Size: 256M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 5.8.2
	Site URL: https://wordpress-647786-2254857.cloudwaysapps.com
	Home URL: https://wordpress-647786-2254857.cloudwaysapps.com
	WP Multisite: No
	Max Upload Size: 256 MB
	Memory limit: 512M
	Permalink Structure: /%year%/%monthnum%/%day%/%postname%/
	Language: en-US
	Timezone: 0
	Debug Mode: Inactive

== Theme ==
	Name: Hello Elementor Child
	Version: 1.0.1
	Author: Elementor Team
	Child Theme: Yes
	Parent Theme Name: Hello Elementor
	Parent Theme Version: 2.4.1
	Parent Theme Author: Elementor Team

== User ==
	Role: administrator
	WP Profile lang: en_US
	User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/95.0.4638.69 Safari/537.36

== Active Plugins ==
	Elementor
		Version: 3.4.7
		Author: Elementor.com

	Elementor Pro
		Version: 3.5.1
		Author: Elementor.com


== Elementor - Compatibility Tag ==
	
	Elementor Pro: Compatible

== Elementor Pro - Compatibility Tag ==
	


== Elements Usage ==
	
	footer : 1
		button : 2
		form : 1
		heading : 4
		icon-list : 1
		image : 1
		nav-menu : 1
	header : 1
		button : 1
		heading : 1
		icon-list : 1
		image : 1
		nav-menu : 1
	wp-page : 6
		animated-headline : 1
		button : 9
		call-to-action : 12
		form : 1
		google_maps : 1
		heading : 19
		icon-list : 3
		shortcode : 1
		testimonial-carousel : 1
		text-editor : 11
		toggle : 2
	not-supported : 0
		button : 3
		heading : 6
		icon-list : 2
		image : 2


== Log ==
	
Log: showing 20 of 272021-10-25 23:42:02 [info] Elementor/Upgrades - _on_each_version Start  
2021-10-25 23:42:03 [info] Elementor/Upgrades - _on_each_version Finished 
2021-10-25 23:42:03 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.4.5',
  'to' => '3.4.6',
)]
2021-10-25 23:42:03 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.4.5',
  'to' => '3.4.6',
)]
2021-11-07 07:45:55 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.4.2',
  'to' => '3.5.0',
)]
2021-11-07 07:45:56 [info] elementor-pro::elementor_pro_updater Started 
2021-11-07 07:45:56 [info] Elementor Pro/Upgrades - _on_each_version Start  
2021-11-07 07:45:56 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2021-11-07 07:45:56 [info] Elementor Pro/Upgrades - _v_3_5_0_price_list Start  
2021-11-07 07:45:56 [info] Elementor Pro/Upgrades - _v_3_5_0_price_list Finished 
2021-11-07 07:45:56 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.4.2',
  'to' => '3.5.0',
)]
2021-11-07 22:26:53 [info] elementor::elementor_updater Started 
2021-11-07 22:26:53 [info] Elementor/Upgrades - _on_each_version Start  
2021-11-07 22:26:53 [info] Elementor/Upgrades - _on_each_version Finished 
2021-11-07 22:26:53 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.4.6',
  'to' => '3.4.7',
)]
2021-11-15 07:14:31 [info] elementor-pro::elementor_pro_updater Started 
2021-11-15 07:14:31 [info] Elementor Pro/Upgrades - _on_each_version Start  
2021-11-15 07:14:31 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2021-11-15 07:14:31 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.5.0',
  'to' => '3.5.1',
)]
2021-11-15 07:14:31 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.5.0',
  'to' => '3.5.1',
)]

PHP: showing 6 of 6PHP: 2021-10-19 11:36:18 [notice X 2][/Users/charlottehough/Documents/Sites/Vernons Solicitors/wp-content/plugins/elementor/includes/managers/image.php::87] Trying to access array offset on value of type bool [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2021-10-19 11:36:20 [error X 2][/Users/charlottehough/Documents/Sites/Vernons Solicitors/wp-content/plugins/elementor-pro/modules/theme-builder/classes/conditions-manager.php::285] Uncaught Error: Call to a member function update_meta() on bool in /Users/charlottehough/Documents/Sites/Vernons Solicitors/wp-content/plugins/elementor-pro/modules/theme-builder/classes/conditions-manager.php:285
Stack trace:
#0 /Users/charlottehough/Documents/Sites/Vernons Solicitors/wp-content/plugins/elementor-pro/modules/theme-builder/classes/conditions-manager.php(166): ElementorPro\Modules\ThemeBuilder\Classes\Conditions_Manager->save_conditions(NULL, Array)
#1 [internal function]: ElementorPro\Modules\ThemeBuilder\Classes\Conditions_Manager->ajax_save_theme_template_conditions(Array, Object(Elementor\Core\Common\Modules\Ajax\Module))
#2 /Users/charlottehough/Documents/Sites/Vernons Solicitors/wp-content/plugins/elementor/core/common/modules/ajax/module.php(171): call_user_func(Array, Array, Object(Elementor\Core\Common\Modules\Ajax\Module))
#3 /Users/charlottehough/Documents/Sites/Vernons Solicitors/wp-includes/class-wp-hook.php(303): Elementor\Core\Common\Modules\Ajax\Module->handle_ajax_request('')
 [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2021-10-21 08:53:05 [notice X 5][/Users/charlottehough/Documents/Sites/Vernons Solicitors/wp-content/plugins/elementor/core/common/modules/ajax/module.php::171] Undefined index: data [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2021-10-28 02:00:45 [error X 1][/Users/charlottehough/Documents/Sites/Vernons Solicitors/wp-content/plugins/elementor/core/wp-api.php::26] Uncaught Error: Call to undefined function Elementor\Core\get_plugins() in /Users/charlottehough/Documents/Sites/Vernons Solicitors/wp-content/plugins/elementor/core/wp-api.php:26
Stack trace:
#0 /Users/charlottehough/Documents/Sites/Vernons Solicitors/wp-content/plugins/elementor/core/wp-api.php(36): Elementor\Core\Wp_Api->get_plugins()
#1 /Users/charlottehough/Documents/Sites/Vernons Solicitors/wp-content/plugins/elementor/modules/system-info/reporters/plugins.php(44): Elementor\Core\Wp_Api->get_active_plugins()
#2 /Users/charlottehough/Documents/Sites/Vernons Solicitors/wp-content/plugins/elementor/modules/system-info/reporters/plugins.php(75): Elementor\Modules\System_Info\Reporters\Plugins->get_plugins()
#3 /Users/charlottehough/Documents/Sites/Vernons Solicitors/wp-content/plugins/elementor/modules/system-info/module.php(298): Elementor\Modules\System_Info\Reporters\Plugins->is_enabled()
#4 /Users/charlottehough/Documents/Sites/Vernons Solicitors/wp-content/plugins/elementor/modules/system-info/module.p [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2021-11-11 22:42:31 [error X 1][/home/647786.cloudwaysapps.com/nadhysntuq/public_html/wp-content/plugins/elementor-pro/modules/theme-builder/classes/conditions-manager.php::286] Uncaught Error: Call to a member function update_meta() on bool in /home/647786.cloudwaysapps.com/nadhysntuq/public_html/wp-content/plugins/elementor-pro/modules/theme-builder/classes/conditions-manager.php:286
Stack trace:
#0 /home/647786.cloudwaysapps.com/nadhysntuq/public_html/wp-content/plugins/elementor-pro/modules/theme-builder/classes/conditions-manager.php(167): ElementorPro\Modules\ThemeBuilder\Classes\Conditions_Manager->save_conditions()
#1 [internal function]: ElementorPro\Modules\ThemeBuilder\Classes\Conditions_Manager->ajax_save_theme_template_conditions()
#2 /home/647786.cloudwaysapps.com/nadhysntuq/public_html/wp-content/plugins/elementor/core/common/modules/ajax/module.php(171): call_user_func()
#3 /home/647786.cloudwaysapps.com/nadhysntuq/public_html/wp-includes/class-wp-hook.php(303): Elementor\Core\Common\Modules\Ajax\Module->handle_ajax_request()
#4 /home/647786.cloudwaysapps.com/nadhysntuq/public_html/wp-includes/class-wp-hook.php(327): WP_Hook->apply_filters()
#5 /home/647786.cloudwaysa [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2021-11-14 09:34:48 [error X 1][/home/647786.cloudwaysapps.com/nadhysntuq/public_html/wp-content/plugins/elementor/core/wp-api.php::26] Uncaught Error: Call to undefined function Elementor\Core\get_plugins() in /home/647786.cloudwaysapps.com/nadhysntuq/public_html/wp-content/plugins/elementor/core/wp-api.php:26
Stack trace:
#0 /home/647786.cloudwaysapps.com/nadhysntuq/public_html/wp-content/plugins/elementor/core/wp-api.php(36): Elementor\Core\Wp_Api->get_plugins()
#1 /home/647786.cloudwaysapps.com/nadhysntuq/public_html/wp-content/plugins/elementor/modules/system-info/reporters/plugins.php(44): Elementor\Core\Wp_Api->get_active_plugins()
#2 /home/647786.cloudwaysapps.com/nadhysntuq/public_html/wp-content/plugins/elementor/modules/system-info/reporters/plugins.php(75): Elementor\Modules\System_Info\Reporters\Plugins->get_plugins()
#3 /home/647786.cloudwaysapps.com/nadhysntuq/public_html/wp-content/plugins/elementor/modules/system-info/module.php(298): Elementor\Modules\System_Info\Reporters\Plugins->is_enabled()
#4 /home/647786.cloudwaysapps.com/nadhysntuq/public_html/wp-content/plugins/elementor/modules/system-info/module.php(255): Elementor [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]


				
```
</details>

**Comment by twerkema** [comment]
Same issue here, I think. It's only happening in my footer template. All other plugins are off and theme is Hello. I added one custom breakpoint for mobile extra. It seems to be the Elementor Advanced and Positioning boxes failing. Fonts/colors still show correctly. Site is a single page in dev mode so I can't share it yet.

When I regenerate css the footer content aligns properly and Margins (30px bottom) and Positioning (width 80%) are applied. When I update the page in the Editor it looks correct but it is wrong on the front end. The margins are gone and Positioning reverts to default. If I add width and margins via custom css it works. Also looks correct when I regenerate css again but as soon as I make an update in the Elementor Editor screen it's gone.

**Comment by Northwest-Studio** [comment]
Same Issue: https://github.com/elementor/elementor/issues/16819

**Comment by DorShahar** [comment]
Merged issue

I merged this issue with  #16819, which is on the same subject so we could deal with it more efficiently. 
Please feel free to continue the conversation there.
Thanks for your feedback!

**Comment by MERZAK-X** [comment]
I faced this issue with many themes footers; changing the breakpoints (+ and -1 then save, just to refresh) fixes it for single lang sites, it's more painful with multi-lang (polylang) sites after fixing one footer in a lang the other glitches. 
Reupdating the footers and regenerating CSS from Elementor dashboard does the trick sometimes.

**Comment by nicholaszein** [comment]
Hello there!

We have great news! 🙌 

### 📢 We're happy to announce that the issue you raised was resolved in Elementor Core v3.15.0! 🥳 

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
